rai_test_utils/rai_test_utils/models/torch/torch_model_utils.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import os
+import urllib.request as request_file
+
+import torch
```

Comment:
I think this code (7-9) should be under try/catch as the import would fail if torch is not installed (not it might be installed as it is an extra.).